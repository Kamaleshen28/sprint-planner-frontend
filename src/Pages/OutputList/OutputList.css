.list-view {
  height: 100%;
  flex-grow: 1;
  /* display: flex;
  justify-content: center; */
  /* align-items: center; */
  background-color: #e4eef1;
}
