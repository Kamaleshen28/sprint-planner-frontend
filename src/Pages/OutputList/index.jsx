import React, { useContext } from 'react';
import { Header, ListView } from '../../Components';
import { DataContext } from '../../Contexts/DataContext';
import './OutputList.css';

export default function OutputList() {
  const { sprints, apiResponse } = useContext(DataContext);
  return (
    <>
      {sprints.length ? (
        <React.Fragment>
          <Header />
          <div className="list-view">
            <ListView heading={apiResponse.title} />
          </div>
        </React.Fragment>
      ) : null}
    </>
  );
}
