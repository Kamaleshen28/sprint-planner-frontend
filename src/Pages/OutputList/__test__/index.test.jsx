import React from 'react';
import DataProvider from '../../../Contexts/DataContext';
import OutputList from '../index';
import { render } from '@testing-library/react';
import {
  sprints as mockSprints,
  apiResponse as mockApiResponse,
} from '../../../mocks';

jest.mock('react-router-dom', () => ({
  useNavigate: () => jest.fn(),
}));

describe('OutputList', () => {
  it('should render correctly', () => {
    const { asFragment } = render(
      <DataProvider
        values={{
          sprints: mockSprints,
          apiResponse: mockApiResponse,
        }}
      >
        <OutputList />
      </DataProvider>,
    );
    expect(asFragment()).toMatchSnapshot();
  });
});
