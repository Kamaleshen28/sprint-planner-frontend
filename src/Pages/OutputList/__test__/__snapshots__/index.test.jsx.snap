// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OutputList should render correctly 1`] = `<DocumentFragment />`;
